modules/ethereumNode.js

```diff
@@ -162,7 +163,7 @@ class EthereumNode extends EventEmitter {
                     Windows.loading.show();
                 })
                 .then(() => {
-                    return this._start(newType || this.type, newNetwork || this.network, newMode || this.nodeMode);
+                    return this._start(newType || this.type(), newNetwork || this.network(), newMode || this.mode());
```

Comment:
I've tested and `this.mode()` was returning a value and `this.mode` wasn't. Are you sure this isn't being made into a function by https://github.com/ethereum/mist/pull/1547/files#diff-409860c680be3333b2089d577f193cf5R65 ?

Will test more this part